modules/bomb_word.py

```diff
@@ -67,23 +127,36 @@ def bomb_word(bot, update, args, job_queue, chat_data):
     word = normalize_text(str(args[0]))[0]
     user_id = update.message.from_user.id
 
-    if 'bombs' not in chat_data:
-        chat_data['bombs'] = {}
-
-    if 'bomb_pidors' not in chat_data:
-        chat_data['bomb_pidors'] = {}
+    initialize_containers(chat_data)
 
     log.debug(chat_data['bombs'])
     if user_id in chat_data['bombs']:
         update.message.reply_text('Ска не еби до завтра!')
     elif len(word) < MIN_LENGTH:
         update.message.reply_text('Слово слишком короткое, как и твой член!')
     else:
-        chat_data['bombs'][user_id] = word
+        chat_data['bombs'][user_id] = {}
+        chat_data['bombs'][user_id]['word'] = word
+        chat_data['bombs'][user_id]['casualties'] = {}
+        chat_data['bombs'][user_id]['expiration_timestamp'] = time.time() + BOMB_TIMEOUT
         chat_data['chat_id'] = update.message.chat_id
         job_queue.run_once(remove_bomb, BOMB_TIMEOUT, context={'bomber': user_id, 'chat_data': chat_data})
         update.message.reply_text('Word bomb has been planted: %s' % word)
 
 
+def bomb_info(bot, update, args, job_queue, chat_data):
+    initialize_containers(chat_data)
+    # reply_payload = json.dumps(chat_data['bombs'])
+    if (not len(chat_data['bombs'])):
```

Comment:
maybe 'len(chat_data['bombs'] > 0' ?

Reply:
see my review

Reply:
I still prefer explicit check for length, your suggestion might also be ok